packages/fxa-auth-server/test/local/routes/account.js

```diff
@@ -6,7 +6,7 @@
 
 const sinon = require('sinon');
 
-const { assert } = require('chai');
+const assert = { ...sinon.assert, ...require('chai').assert };
```

Comment:
Oh nice. As you're doing this more than once, should we pull it into in a little `test/assert.js` module just to eliminate the repetition?